cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -402,31 +381,8 @@ def check_index_page(self, separate_archived_courses, org):
         parsed_html = lxml.html.fromstring(index_response.content)
         course_tab = parsed_html.find_class('courses')
         self.assertEqual(len(course_tab), 1)
-        course_links = course_tab[0].find_class('course-link')
```

Comment:
I had to remove the checks about specific courses, since the listings are client-side rendered by React. However, I left whether or not the top-level "archived" tab appears, since that is still server-side rendered.

I covered which list the courses appear in with bok choy tests (below).